packages/remote-config/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/remoteconfig/FirebaseRemoteConfigPlugin.java

```diff
@@ -156,7 +166,26 @@ public void getString(PluginCall call) {
 
     @PluginMethod
     public void setMinimumFetchInterval(PluginCall call) {
-        call.reject("Not available on Android.");
+        call.reject("Not available on Android. Use setConfigSettings.");
+    }
+
+    @PluginMethod
+    public void setConfigSettings(PluginCall call) {
+        try {
+            Integer fetchTimeoutInSeconds = call.getInt("fetchTimeoutInSeconds", DEFAULT_FETCH_TIMEOUT_IN_SECONDS);
+            Integer minimumFetchIntervalInSeconds = call.getInt("minimumFetchIntervalInSeconds", DEFAULT_MINIMUM_FETCH_INTERVAL_IN_SECONDS);
+
+            implementation.setConfigSettings(fetchTimeoutInSeconds, minimumFetchIntervalInSeconds)
+                .addOnCompleteListener(t -> {
+                    // NOTE: Android remote config sdk does not have an interface to get setting value.
+                    // Therefore, the minimumFetchIntervalInSeconds is stored in a property of this class.
+                    this.customMinimumFetchIntervalInSeconds = minimumFetchIntervalInSeconds;
```

Comment:
This plugin is only a wrapper for the official SDK. If the official SDK does not offer this functionality, then this plugin does not offer it either. Please remove it.

Reply:
@robingenz Thanks, I understand your thoughts.

Let me explain my intention. I figured that if a developer set `minimumFetchIntervalInSeconds` in `setSettings`, the developer would assume that `fetchConfig` would also use that setting internally.

Since the Android SDK could not retrieve the setting value, I stored it in `customMinimumFetchIntervalInSeconds`.

However, the SDK reference showed that there was a `fetch()` method apart from `fetch(longminimumFetchIntervalInSeconds)`. It internally uses the `minimumFetchIntervalInSeconds` set by `setSettings`. If it is not set, then the default value of 12 hours is used.

So I’ll try to remove `customMinimumFetchIntervalInSeconds` and use `fetch()`.

If you have any suggestions, please let me know.

Reply:
I have tried this. https://github.com/capawesome-team/capacitor-firebase/pull/694/commits/1e1d59ecefb722ee1bdf60dccc3dc328e9a7470e

However, in my testing, I found that in some cases the behavior was not as expected on Android. Because the values set in setSettings are stored in SharedPreferences on Android, so the default values were not used.

So, I will revert the fetchConfig code and write in a comment that the setSettings values are not used in fetchConfig.

Reply:
Sorry, the above behavior was the same for fetchAndActivate, so I will take it as the AndroidSDK specification and add a supplementary comment without changing the fetchConfig code.